apps/test/unit/templates/sectionsRefresh/SectionsSetUpContainerTest.jsx

```diff
@@ -107,10 +110,47 @@ describe('SectionsSetUpContainer', () => {
 
     expect(fetchSpy).to.have.been.called.once;
 
+    await new Promise(resolve => setTimeout(resolve, 0));
+    expect(navigateToHrefSpy).to.have.been.called.once;
+    expect(navigateToHrefSpy.getCall(0).args[0]).to.include('/home');
+
+    sinon.restore();
+  });
+
+  it('appends showSectionCreationDialog to url if isUsersFirstSection is true', async () => {
```

Comment:
This worked locally before I merged staging in. I kept these commented out as similar tests were commented out

Reply:
My [pr here](https://github.com/code-dot-org/code-dot-org/pull/51676) will relaunch these tests!